src/main/java/de/uhd/ifi/se/decision/management/jira/extraction/classification/ClassificationManagerForCommentSentences.java

```diff
@@ -77,8 +77,8 @@
 		for (Comment comment : commentsList) {
 			for (Sentence sentence : comment.getSentences()) {
 				if (sentence.isRelevant() && sentence.isTaggedFineGrained()) {
-					sentence.setKnowledgeTypeString(
-							ActiveObjectsManager.getElementFromAO(sentence.getId()).getKnowledgeTypeString());
+					Sentence aosentence = (Sentence) ActiveObjectsManager.getElementFromAO(sentence.getId());
+					sentence.setKnowledgeTypeString(aosentence.getKnowledgeTypeString());
```

Comment:
Please use the same names as in DecisionKnowledgeElement: `void setType(String type);`